### PR TITLE
WAS-1011 Add visually-hidden content to 'change this' on summary page

### PR DESCRIPTION
**Pull request by woodpigeon**

### CODE REVIEW

File: spec/features/reviewing_spec.rb [hunk 1]
```diff
@@ -21,6 +21,21 @@
...
+        ids_of_inaccessible_change_links = inaccessible_change_links.map {|p| p[:id]}
+
+        expect(inaccessible_change_links.count).to eq(0),
+          "Change links with missing accessible content: #{ids_of_inaccessible_change_links.join(',')}"
+      end
+    end
+  end
+
   context "Check details basics and Farming" do
     let(:enrollment_review) { DigitalServicesCore::EnrollmentReview.new(page_object.enrollment) }
```
Comment (autotelik) [inline spec/features/reviewing_spec.rb]: I'd be temtpted to add  calls like this as helper methods in the page object  as potential useful across multiple specs

        all_change_links = page.all("form tr td a.change-link")

support/page_objects/reviewing_po.rb

Comment (woodpigeon) [inline spec/features/reviewing_spec.rb]: @autotelik Good call, thanks

**Comment by timstone** [comment]
QA passed

**Comment by timstone** [comment]
QA passed

**Pull request merged**